archive/c/c-plus-plus/reverse-string.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+#include <string>
+#include <algorithm>
+
+int main()
+{
+    std::string s = "Some arbitrary text";
```

Comment:
Can you pick up this line of text from the command line?

Reply:
Sure. Fixed